<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebScraping Steam </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_Categorias">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>411</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_Juegos">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>441</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>482</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
